pkg/detectors/databrickstoken/databrickstoken.go

```diff
@@ -21,64 +20,66 @@ var (
 	client = common.SaneHttpClient()
 
 	// Make sure that your group is surrounded in boundary characters such as below to reduce false positives.
-	domain = regexp.MustCompile(`\b(https:\/\/[a-z0-9-]+\.cloud\.databricks\.com)\b`)
-	keyPat = regexp.MustCompile(`\b(dapi[a-z0-9]{32})\b`)
+	keyPat    = regexp.MustCompile(`\b(dapi[0-9a-f]{32})(-\d)?\b`)
```

Comment:
@4390c336 we're gonna prioritize merging https://github.com/trufflesecurity/trufflehog/pull/1965/files since it was opened first. The most important change is expanding the `domain` which both PRs include. However, I'm curious about this change to `keyPat`. Could you explain the reason behind this change? Do all databricks tokens have an appened `-{digit}`?

cc @ankushgoel27 (you don't need to do anything atm besides respond to the changes I requested in your PR, but wanted to make you aware of this one too)

Reply:
@zricethezav as indicated by regex . the `-{digit}` part is optional. not all tokens have it. some of them do.

source : I work for Databricks, and we have this running to scan our internal/public repos.

Reply:
PS : i don't care about the competition or credit as long as we merge a working detector

Reply:
@zricethezav let me know if you would like to add the regex with optional digit added to my PR.

Reply:
@ankushgoel27, yes please. I'll defer to @4390c336 as he is an engineer at Databricks.